src/classes/BDI_DataImportService_TEST.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>37.0</apiVersion>
```

Comment:
@force2b , the new class has the version as the BDI_DataImport2_TEST from which a unit test has been moved into. Shall I change it to version 42.0?

Reply:
This should go to v43 since it's a new class.